docs/further.md

```diff
@@ -618,6 +618,14 @@ This configuration directs SLURM logs to a centralized location, making them eas
 Running Snakemake within an active SLURM job can lead to unpredictable behavior, as the execution environment may not be properly configured for job submission.
 To mitigate potential issues, the SLURM executor plugin detects when it's operating inside a SLURM job and issues a warning, pausing for 5 seconds before proceeding.
 
+### Running in SLURM reservations
+
+This scenario is mostly done when testing or teaching.
+The plugin allows specifying a flag `--slurm-reservation=<name>` to use a particular reservation.
+It does not validate the spelling nor eligibility to this reservation.
+When teaching, it is recommended that you ask your administrators to set up a reservation with the SLURM `magnetic` flag.
+This will ensure that the `--slurm-reservation` flag is not necessary, e.g. if the reservation is associated with a particular course account. 
```

Comment:
I feel that the description in the plugin should be a bit more objective since users might use it under all kinds of situations. Even though your suggestions relative to teaching are relevant, I think they might be better placed somewhere else, for example in the [HPC teaching material](https://github.com/snakemake/snakemake-hpc-teaching-material).

```suggestion
The plugin allows specifying a flag `--slurm-reservation=<name>` to use a particular reservation.
It does not validate the spelling nor eligibility to this reservation.
```

Reply:
agreed.